city_scrapers/spiders/il_police_professionalism.py

```diff
@@ -72,17 +96,13 @@ def _parse_all_day(self, item):
     def _parse_location(self, item):
         """Parse or generate location."""
         return {
-            "address": "",
-            "name": "Commission on Police Professionalism",
+            "address": "301 S 2nd St, Springfield, IL 62701",
+            "name": "Illinois State Capitol",
         }
 
     def _parse_links(self, item):
         """Parse or generate links."""
-        link = item.xpath('@href').get()
-        return [{
-            "href": "https://www.isp.state.il.us/media/" + link,
-            "title": "Commission on Police Professionalism"
-        }]
+        return [{"href": "", "title": "Commission on Police Professionalism"}]
```

Comment:
Assuming this is a placeholder for now? This is a weird one because the agenda is also the source link, but since that's the case you can return it that way

```python
def _parse_links(self, response):
    return [{"href": response.url, "title": "Agenda"}]
```